src/modules/components/proposal/proposalActions/proposalActions/proposalActions.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/react';
 
+import { generateProposalActionTokenMint } from '../actions/generators/proposalActionTokenMint';
```

Comment:
@cgero-eth should we barrel export generators? I feel they are for internal use only right?

Reply:
Yes we could, at the end these are only used on tests or storybook and not included to the library bundle